docs/vyper-by-example.rst

```diff
@@ -275,14 +275,14 @@ the funds will be sent to the  beneficiary at the end of the campaign deadline.
 Participants will be refunded their respective contributions if the total
 funding does not reach its target goal.
 
-.. literalinclude:: ../examples/crowdfund.vy
+.. literalinclude:: ../examples/crowdfund/crowdfund.vy
```

Comment:
You know, I actually prefer all the examples to not be inside each of these folders, don't think it adds anything.